src/RadioButtons/index.stories.js

```diff
@@ -17,7 +17,7 @@ export const Example = () => {
   const [value, setValue] = useState();
   return (
     <div className="nds-typography">
-      <h3> What is your favourite colour? </h3>
+      <h3 style={{marginBottom: "16px"}}> What is your favourite colour? </h3>
```

Comment:
Good use case for a helper class:
```suggestion
      <h3 className="margin--bottom--l"> What is your favourite colour? </h3>
```

Reply:
Accessible through https://github.com/narmi/design_system/blob/fix-radiobuttons-cleanup/src/helper-classes/spacing.scss 👍

Reply:
accessible through https://github.com/narmi/design_system/blob/fix-radiobuttons-cleanup/src/helper-classes/spacing.scss 👍